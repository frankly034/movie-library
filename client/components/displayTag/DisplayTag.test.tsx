import { render } from "@testing-library/react";

import DisplayTag from ".";

describe("Component - DisplayTag", () => {
  it("should renders the component", () => {
    const { container } = render(<DisplayTag>Horror</DisplayTag>);
    expect(container).toMatchSnapshot();
  });

  it("should render text", () => {
    const { getByText } = render(<DisplayTag>Documentary</DisplayTag>);
    expect(getByText("Documentary")).toBeTruthy;
  });
});
