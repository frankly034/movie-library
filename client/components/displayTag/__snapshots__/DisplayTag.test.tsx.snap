// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component - DisplayTag should renders the component 1`] = `
<div>
  <span
    class="sc-bcPKhP djlObm"
  >
    Horror
  </span>
</div>
`;
